src/pages/Main/MainSlide.js

```diff
@@ -1,16 +1,28 @@
 import React from 'react';
+import { useNavigate, useParams } from 'react-router-dom';
 import Slider from 'react-slick';
 import styled from 'styled-components';
 import 'slick-carousel/slick/slick.css';
 import 'slick-carousel/slick/slick-theme.css';
 
 const MainSlide = () => {
+  const navigate = useNavigate();
+  const param = useParams();
+  const userId = param.user_id;
+  const writingId = param.writing_id;
```

Comment:
다음처럼 구조분해할당해서 사용하실 수 있습니다!
```suggestion
  const { user_id, writing_id } = useParams();
```
그리고 naming convention때문에 변환하시는 거라면, Router에서 params 자체의 naming을 camelCase로 바꿔주시면 될 것 같네요!